ast/io/ast_reader.h

```diff
@@ -0,0 +1,59 @@
+#ifndef ASTREADER
+#define ASTREADER
+
+#include "../ast.h"
+#include <stdbool.h>
+
+struct AST_Whole_Program* readAST(char* filename, bool debug);
+
+// -----------------------
```

Comment:
same alignment of the indentation would look better. so all function names can be read from top to bottom.

Reply:
fixed in d491cbd7